L/LVGL_SDL/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LVGL_SDL"
+version = v"9.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/lvgl/lvgl.git", "4f086111a107718b719a45e2cc422f5b756d7edd"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/lvgl
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/*.patch
+if [[ "${target}" == "x86_64-apple-darwin14" ]]
+then
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/x86_64-apple-darwin14/*.patch
+fi
+cp lv_conf_template.h lv_conf.h
+cp lv_conf.h ..
+
+cmake -B build \
+      -DBUILD_SHARED_LIBS=ON \
+      -DLIB_INSTALL_DIR=${prefix}/lib \
+      -DLV_CONF_BUILD_DISABLE_DEMOS=1 \
+      -DLV_CONF_BUILD_DISABLE_EXAMPLES=1 \
+      -DLV_CONF_INCLUDE_SIMPLE=OFF \
+      -DCMAKE_INSTALL_PREFIX=${prefix} \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release
+cmake --build build --parallel ${nproc}
+cmake --install build
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "linux"; libc="glibc")
```

Comment:
This is a very odd list of platforms. Why does it only build on these two specific ones?

Reply:
These are the only two platforms that I am able to test (my dev laptop and my embedded target board).

Reply:
That's not a great reason to limit the list: it's not expected that you personally test all platforms. But by opening up, you vastly expand who can benefit from this JLL; and if anyone uses them, and there are problems, then they may give you feedback or may even help out.

Of course if other platforms cause build errors that are hard to track down, it's OK to remove them and worry about those you need first. But looking at this list I'd be surprised if not at least Linux x86 and mac ARM would work, too.

So how about starting with the full list, and see if any build jobs fail here on Yggdrasil, and then we can see how to proceed :-)